app/controllers/item.js

```diff
@@ -38,9 +37,9 @@ export default Ember.Controller.extend({
   }),
 
   hasDraftOrder: Ember.computed.alias("session.draftOrder"),
-  isOrderFulfilmentUser: Ember.computed(function() {
+  isOrderFulfilmentUser: Ember.computed('session.authToken', function() {
     let user = this.get('session.currentUser');
-    return user.hasRole('Order fulfilment');
+    return user && user.hasRole('Order fulfilment');
```

Comment:
Unnecessary conditions

Reply:
If this is executing after logout than this condition is required. Because it tries to fetch `roles` from `user`.

Reply:
Computed property shouldn't get called if no user exist